6213 Бахтин Матей Павлович Лаб.2 Вар.2

xe-n0x opened the pull request:
**ФИО:** Бахтин Матвей
**Номер группы:** 6213 
**Номер лабораторной:** 2  
**Номер варианта:** 01  
**Краткое описание решаемой задачи:** Согласно варианту 01 лабораторной 2 мне надо было скачать изображения по ключевому слову "cat".  
**Краткое описание работы кода:** Модуль `lab2.py` скачивает от 50 до 1000 изображений по ключевому слову "cat", создаёт аннотацию с путями к скачанным картинкам, он же и является точкой входа в приложение. Чтобы его запустить, требуется указать аргументы командной строки: `python lab2.py --min_width --max_width  --min_height --max_height  --output_folder --csv_path`

xe-n0x committed:
готовая ЛР 1

xe-n0x committed:
Update lab1.py

xe-n0x committed:
Сделал lab2

Готовая lab2

vfkon commented on lab2.py:
```diff
@@ -0,0 +1,89 @@
+        writer.writerow(['absolute path', 'relative path'])
+        for filename in os.listdir(output):
+          abs_path = os.path.abspath(os.path.join(output, filename))
+          rel_path = os.path.join(output, filename)
```
это не относительный путь

vfkon reviewed (changes requested):
Исправьте относительный путь, изучите что это такое и каким образом строится относительный путь

xe-n0x committed:
lab3_var1

Сделал 3 лабу, немного изменил структуру своих заданий на гите, разбив по папкам лабы

xe-n0x committed:
Delete lab3.py
